src/services/healthcareProfessionalService.ts

```diff
@@ -55,51 +55,67 @@ export async function addHealthcareProfessional(
     }
 }
 
-export async function addHealthcareProfessionalToFacility(facilityId: string, 
-    healthcareProfessional: gqlTypes.HealthcareProfessionalInput) {
+/**
+ * Creates a HealthcareProfessional and adds it to the listed facilities
+ * @param healthcareProfessionalInput 
+ * @returns A HealthcareProfessional object
+ */
+export async function addHealthcareProfessionalToFacility( 
```

Comment:
I think this function might actually be no longer necessary. Let's talk through it on slack